preset/rabbitmq/preset.go

```diff
@@ -22,6 +26,13 @@ const defaultVersion = "alpine"
 const defaultPort = 5672
 const managementPort = 15672
 
+// Message is a single message sent to RabbitMQ.
+type Message struct {
+	Queue       string `json:"queue"`
+	ContentType string `json:"contentType"`
```

Comment:
```suggestion
	ContentType string `json:"content_type"`
```
To align with other JSON tags in this project😼